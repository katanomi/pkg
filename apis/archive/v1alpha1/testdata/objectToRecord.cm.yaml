apiVersion: v1
data:
  secret: b
kind: ConfigMap
metadata:
  creationTimestamp: "2022-12-29T03:49:23Z"
  name: abc
  namespace: default
  resourceVersion: "87567012"
  uid: 516f7695-3c3c-475f-bbaa-ee8db0d1f77d
