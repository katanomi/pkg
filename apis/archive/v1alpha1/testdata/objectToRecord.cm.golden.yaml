spec:
  uid: 516f7695-3c3c-475f-bbaa-ee8db0d1f77d
  group: ""
  version: v1
  kind: ConfigMap
  namespace: default
  name: abc
  creationTimestamp: 1672285763
  data:
    apiVersion: v1
    data:
      secret: b
    kind: ConfigMap
    metadata:
      creationTimestamp: "2022-12-29T03:49:23Z"
      name: abc
      namespace: default
      resourceVersion: "87567012"
      uid: 516f7695-3c3c-475f-bbaa-ee8db0d1f77d
